docs/docs/rest-api/public/api/v2/types/label.raml

```diff
@@ -0,0 +1,10 @@
+#%RAML 1.0 DataType
+type: object
+properties:
+  name:
+    type: string
+    minLength: 1
+    maxLength: 255
+    pattern: (?!^((MARATHON_)|(DCOS_)|(MESOS_)).*)
```

Comment:
i like the pattern. but it stops people from using special labels that we come up with. "reserved for internal use" was perhaps the wrong statement. maybe "reserved for use by marathon, mesos, and/or dcos" would have been better? basically don't want user's inventing arbitrary labels that they craft names for. but we may tell them to set some special MARATHON_XYZ label to accomplish something